### PR TITLE
Add Elasticsearch index templates

### PR DESCRIPTION
**Pull request by gthvn1**
This patch adds three index templates that are used by StackLight.

### CODE REVIEW

**Review by simonpasquier — APPROVED** [review]

File: elasticsearch/files/es_template_kibana4.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+          "uiStateJSON" : {
+            "type" : "string"
+          },
+          "version" : {
+            "type" : "integer"
+          }
+        }
+      }
+    },
+    "template": "Kibana4"
```
Comment (SwannCroiset) [inline elasticsearch/files/es_template_kibana4.json]: "template" must be a pattern matching the name of indices. like log-*
here the template is ".kibana"

File: elasticsearch/files/es_template_log.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            "analyzer": {
+                 "ost_roles": {
+                     "type": "custom",
+                     "tokenizer": "comma",
+                     "filter": ["lowercase"]
+                 }
+            }
+        }
+    },
+    "template": "Logs"
```
Comment (SwannCroiset) [inline elasticsearch/files/es_template_log.json]: log-*

File: elasticsearch/files/es_template_notification.json [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            "analyzer": {
+                 "ost_roles": {
+                     "type": "custom",
+                     "tokenizer": "comma",
+                     "filter": ["lowercase"]
+                 }
+            }
+        }
+    },
+    "template": "Notifications"
```
Comment (SwannCroiset) [inline elasticsearch/files/es_template_notification.json]: notification-*

**Review by SwannCroiset — CHANGES_REQUESTED** [review]

**Comment by gthvn1** [comment]
I have updated the review and added the possibility to delete an index.

**Review by elemoine — APPROVED** [review]

**Review by simonpasquier — APPROVED** [review]

**Review by SwannCroiset — APPROVED** [review]

**Pull request merged**